test/integration/acceptance/get_account_test.cpp

```diff
@@ -0,0 +1,247 @@
+  }
+
+  /**
+   * Creates valid GetAccount query of current user
+   * @param hash of the tx for querying
+   * @return built query
+   */
+  auto makeQuery(const std::string &user) {
+    return complete(baseQry().queryCounter(1).getAccount(user));
+  }
+
+  auto makeQuery() {
+    return makeQuery(kUserId);
+  }
+
+  /**
+   * @return a lambda that verifies that query response says the query has
+   * no account at response
+   */
+  auto checkNoAccountResponse() {
+    return [](auto &response) {
+      ASSERT_TRUE(boost::apply_visitor(
+          shared_model::interface::QueryErrorResponseChecker<
+              shared_model::interface::NoAccountErrorResponse>(),
+          response.get()))
+          << "Actual response: " << response.toString();
+    };
+  }
+
+  /**
+   * @return a lambda that verifies that query response contains created account
+   */
+  auto checkValidAccount(const std::string &domain,
+                         const std::string &user,
+                         const std::string &role) {
+    return [&](const proto::QueryResponse &response) {
+      ASSERT_NO_THROW({
+        const auto &resp = boost::apply_visitor(
+            framework::SpecifiedVisitor<interface::AccountResponse>(),
+            response.get());
+        ASSERT_EQ(resp.account().accountId(), user);
+        ASSERT_EQ(resp.account().domainId(), domain);
+        ASSERT_EQ(resp.roles().size(), 1);
+        ASSERT_EQ(resp.roles()[0], role);
+      }) << "Actual response: "
+         << response.toString();
+    };
+  }
+
+  auto checkValidAccount() {
+    return checkValidAccount(kDomain, kUserId, kNewRole);
+  }
+
+  auto checkStatefulInvalid() {
+    return [](auto &status) {
+      ASSERT_NO_THROW(boost::apply_visitor(
+          shared_model::interface::QueryErrorResponseChecker<
+              shared_model::interface::StatefulFailedErrorResponse>(),
+          status.get()));
+    };
+  }
+
+  auto checkStatelessInvalid() {
+    return [](auto &status) {
+      ASSERT_NO_THROW(boost::apply_visitor(
+          shared_model::interface::QueryErrorResponseChecker<
+              shared_model::interface::StatelessFailedErrorResponse>(),
+          status.get()));
+    };
+  }
+
+  const std::string kNewRole = "rl";
+  const std::string kRole2 = "roletwo";
+  const std::string kUser2 = "usertwo";
+  const crypto::Keypair kUser2Keypair =
+      crypto::DefaultCryptoAlgorithmType::generateKeypair();
+};
+
+#define check(i) [](auto &block) { ASSERT_EQ(block->transactions().size(), i); }
+
+/**
+ * @given a user with GetMyAccount permission
+ * @when GetAccount is queried on the user
+ * @then there is a valid AccountResponse
+ */
+TEST_F(GetAccount, Basic) {
+  IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTxAwait(makeUserWithPerms(), check(1))
+      .sendQuery(makeQuery(), checkValidAccount())
+      .done();
+}
+
+/**
+ * @given a user with all required permissions
```

Comment:
There's might be inconsistency in future. What's the point of that?

Reply:
It was discovered that it was required from the beginning. I suggest writing such titles only for those cases that were mentioned in the Jira task. Without them, it would be much harder to identify test cases on [this page](https://soramitsu.atlassian.net/wiki/spaces/IS/pages/299958273/Acceptance+Tests).